app/src/renderer/vuex/modules/send.js

```diff
@@ -265,3 +271,44 @@ function handleSDKError(err) {
   }
   throw new Error(message)
 }
+
+function getErrorMessage(code) {
```

Comment:
I think would be nice to have this out of any particular module, to have it usable by others